contracts/Storage/Structs.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.6.9;
+
+
+contract Structs {
```

Comment:
why are you doing a separate contract for structs and storage?

Reply:
"All structs should be removed from Storage.sol. If any of these structs are needed in Storage.sol, they must also be removed from their original contract and placed into a separate Structs.sol. If needed, import Structs.sol into Storage.sol and other dependent contracts."
this is recommended by contributors of eip1822